.github/workflows/format.yaml

```diff
@@ -1,13 +1,15 @@
 name: Formatting Checks
 on:
-  pull_request:
+  push:
     branches:
       - main
       - develop
-  push:
+      - sims2cloud
```

Comment:
Do we anticipate having a public sims2cloud branch?

Reply:
Not really, this is meant for the private project! One should keep in mind it is harmless to add private branches here. But when this is synced with `main` of ft-stack, and then people sync `sims2cloud` with that main, it means tests will be properly executed on the private branch.